src/main.rs

```diff
@@ -217,7 +218,7 @@ struct EmptyMetadata {}
 
 #[derive(Debug, Deserialize)]
 struct DynamoOutputGraphMetadata {
-    sizes: Option<FxHashMap<String, Vec<SymInt>>>,
+    _sizes: Option<FxHashMap<String, Vec<SymInt>>>,
```

Comment:
It will change the field name - so if you need it to match the JSON name then you can prefix it with:
```
#[allow(dead_code)]
sizes: Option<FxHashMap<String, Vec<SymInt>>>,
```
so it doesn't complain about the unread field.

Reply:
ok, i'll need to do this. fortunately these are all defaulted so it doesn't matter too much yet